src/components/InventoryGroups/Modals/CreateGroupModal.js

```diff
@@ -7,6 +7,15 @@ import { createGroup, validateGroupName } from '../utils/api';
 import { useDispatch } from 'react-redux';
 import awesomeDebouncePromise from 'awesome-debounce-promise';
 
+export const validate = async (value) => {
+  const results = await validateGroupName(value.trim());
+  if (results === true) {
+    throw 'Group name already exists';
+  }
+
+  return undefined;
```

Comment:
Question: why would we need to return undefined explicitly?

Reply:
This is the requirement of data-driven-forms. In order to mark that the validator succeeds, it shouldn't return nothing or just undefined 😄